DakshChan/IBS
# [IBS-22] Submit Marks Through UI

[Ticket](https://ibs-utm.atlassian.net/browse/IBS-22?atlOrigin=eyJpIjoiYzM3NmVmODkxMmZiNGFjOThjZjMzODE4OTA1OTdhYjQiLCJwIjoiaiJ9)

## Testing
- Create a course
- Add instructor to course
- Add student to course
- Create new task for the course
- Add criteria for the task
- Go to either the view grades page or submit marks page
- Submit marks for a criteria
- The table should update automatically
